pkg/module/scaffold/init.go

```diff
@@ -0,0 +1,84 @@
+package scaffold
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/mandelsoft/vfs/pkg/cwdfs"
+	"github.com/mandelsoft/vfs/pkg/vfs"
+)
+
+const directoryMode os.FileMode = 0755
+
+type resourcesToCreate struct{}
```

Comment:
This type is redundant. Just having the unexported function `start(rb resourceBuilder) error` without a receiver would achieve the same result in `createResources()`.

Probably you can then just merge `start()` and `createResources()` into a single function.

Reply:
👍